.github/workflows/test_functional.yml

```diff
@@ -56,34 +60,34 @@ jobs:
             platform: '_local_background*'
           # tests/k
           - name: 'flaky'
-            os: 'ubuntu-22.04'
+            os: 'ubuntu-latest'
             python-version: '3.7'
             test-base: 'tests/k'
             chunk: '1/1'
             platform: '_local_background* _local_at*'
           # remote platforms
           - name: '_remote_background_indep_poll'
-            os: 'ubuntu-22.04'
+            os: 'ubuntu-latest'
             python-version: '3.7'
             test-base: 'tests/f tests/k'
             chunk: '1/1'
             platform: '_remote_background_indep_poll _remote_at_indep_poll'
           - name: '_remote_background_indep_tcp'
-            os: 'ubuntu-22.04'
+            os: 'ubuntu-latest'
             test-base: 'tests/f tests/k'
             python-version: '3.7'
             chunk: '1/1'
             platform: '_remote_background_indep_tcp _remote_at_indep_tcp'
           # macos
           - name: 'macos 1/5'
             os: 'macos-latest'
-            python-version: '3.9'
+            python-version: '3.8'  # oldest available
             test-base: 'tests/f'
             chunk: '1/5'
             platform: '_local_background*'
           - name: 'macos 2/5'
             os: 'macos-latest'
-            python-version: '3.9'
+            python-version: '3'  # newest available
```

Comment:
The original intention was to run the `1/4` chunk in MacOS on the oldest Python version. (However at some point this was split into `1/5` and `2/5` for faster completion.)

What is the intention behind running different chunks in different Python versions?

Reply:
I thought we should probably have a test for latest Python on Mac OS as a canary. Happy to revert (unrelated really).